assets/models/Map.js

```diff
@@ -101,6 +101,12 @@ class Map {
 
   filterMarkerCount(subCategoryIdArr = []) {
     let count = 0;
+    
+    // when no filter option is applied give count of all reports
+    if (subCategoryIdArr.length === 0) {
+      return this.mapLayers.getLayers().length;
+    }
+
```

Comment:
when no filter is applied, this gets the count of the markers on the map - only hazard markers